thunder/executors/nvfuserex_impl.py

```diff
@@ -161,8 +161,8 @@ def is_supported_devicetype(devicetype: DeviceType) -> bool:
 
 
 def device_supports_fp8() -> bool:
-    cuda_major, _ = torch.cuda.get_device_capability()
-    return cuda_major > 8
+    cuda_major, cuda_minor = torch.cuda.get_device_capability()
+    return (cuda_major, cuda_minor) >= (8, 9)
```

Comment:
I realize that this is an ugly bit...

I think the full logic here should copy this: https://github.com/NVIDIA/Fuser/blob/6fa084312d7eec5c69d59f3eb3cbdd9fa72a1600/csrc/device_lower/analysis/device_version.cpp#L24-L39

But that's a lot... We should have a generic API on nvfuser side that does `is_dtype_support_on_device(dtype, device_index)`

Reply:
A Python function exposed by nvFuser for this logic would be great!

Reply:
(Even if we don't do that in this PR, an issue for it would be great)